<launch>
    <node pkg="aist" type="camera_node" name="camera_node" respawn="true" output="screen">
        <param name="device_id" value="0" />
        <param name="framerate" value="30" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="output_image_topic" value="/clean_image/image_raw" />
        <param name="calibration_file_url" value="$(find aist)/assets/calib.xml" />
    </node>
</launch>